city_scrapers/spiders/chi_board_elections.py

```diff
@@ -0,0 +1,208 @@
+            yield from self._prev_meetings(response)
+        else:
+            yield from self._next_meeting(response)
+
+    def _next_meeting(self, response):
+        next_meeting = response.xpath('//div[@class="copy"]/text()').extract()[2]
+        meetingdate = re.search(r'\d{1,2}:.*\d{4}', next_meeting).group(0)
+        data = {
+            '_type': 'event',
+            'name': "Electoral Board",
+            'event_description': "",
+            'classification': COMMISSION,
+            'start': self._parse_start(meetingdate, next_meeting),
+            'all_day': False,
+            'documents': [],
+            'sources': [{
+                'url': response.url,
+                'note': ''
+            }],
+        }
+        if "69" in next_meeting:
+            data['location'] = self._parse_location(response)
+        else:
+            raise ValueError("The address has changed.")
+
+        data['end'] = {
+            'date': data['start']['date'],
+            'time': None,
+            'note': '',
+        }
+
+        link = response.xpath("//a/@href").extract()[2]
+        data['documents'] = self._parse_documents(link)
+        data['status'] = self._generate_status(data)
+        data['id'] = self._generate_id(data)
+
+        yield data
+
+    def _prev_meetings(self, response):
+        """
+        Meetingdate regex first searches for the 3 types of hyphens that chi_board_elections
+        uses (they like switching it up), and then finds a year number,
+        and returns everything in between.
+        """
+
+        meetings = response.xpath("//a").extract()
+        prevdate = None
+        for meeting in meetings:
+            meeting.replace('\xa0', ' ')  # Gets rid of non-breaking space character
+            try:
+                meetingdate = re.search(r'(–|- |-)(.+[0-9]{4})', meeting).group(2)
+                while len(meetingdate) > 30:
+                    meetingdate = re.search(r'(–|- |-)(.+[0-9]{4})', meetingdate).group(2)
+                meetingdate.lstrip()
+                if prevdate != meetingdate:  # To acount for duplicates
+                    data = {
+                        '_type': 'event',
+                        'name': "Electoral Board",
+                        'event_description': "",
+                        'classification': COMMISSION,
+                        'start': self._parse_start(meetingdate, meeting),
+                        'all_day': False,
+                        'location': self._parse_location(response),
+                        'documents': [],
+                        'sources': [{
+                            'url': response.url,
+                            'note': ''
+                        }],
+                    }
+                    data['end'] = {
+                        'date': data['start']['date'],
+                        'time': None,
+                        'note': '',
+                    }
+                    if "Minutes" in meeting:
```

Comment:
This can be moved into the `_parse_documents` method, and we should pull the links to the videos as well. You should be able to generalize it by checking if a given node is a link tag, and then using the first part of the string before the dash (so "Board Meeting Minutes - ..." becomes "Board Meeting Minutes). You could do that with something like

```python
meeting_str.split(' - ')[0]
```